packages/cli/lib/developerTestAccounts.js

```diff
@@ -0,0 +1,14 @@
+const {
+  HUBSPOT_ACCOUNT_TYPES,
+} = require('@hubspot/local-dev-lib/constants/config');
+
+const DEV_TEST_ACCOUNT_STRING = 'dev test account';
```

Comment:
@markhazlewood let me know your thoughts on this naming - I posted some screenshots in the main PR body

Reply:
I'll defer to Mark here for final say, but should we be consistent with the use of `dev` and `developer` in the account names (i.e. `dev test account` vs. `developer sandbox`)? I'm partial to the shorter `dev` naming.

Reply:
Agree. For external consumption we should always use "developer test account" fully spelled out.

Reply:
Cool - will update to the full name 👍  thanks!

Reply:
One more question for you @markhazlewood42 - do we want to differentiate between standard accounts (customer portals) and app developer accounts? Like do we want to show `[app developer]` next to those accounts